src/Enums/SharedType.java

```diff
@@ -0,0 +1,18 @@
+package Enums;
+
+public enum SharedType {
+    PHONE {
+        @Override
+        public String share() {
+            return "Ticket is shared via phone";
+        }
+    },
+    PHONE_EMAIL {
+        @Override
+        public String share() {
+            return "Ticket is shared via phone and email";
+        }
+    };
+
+    public abstract String share();
```

Comment:
Enums needs for represent a group of constants, don't mix it with business logic like share() method